collections/_infographics/emissions/emissions-eu-trends.md

```diff
@@ -13,4 +13,38 @@ data-orig:
   - [ "Eurostat", "https://ec.europa.eu/eurostat/web/products-datasets/-/ENV_AIR_GGE" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emise-eu-vyvoj" %}
+## How to read this graph
+
+* All emission values are given in units of <glossary id="co2eq">CO<sub>2</sub>eq</glossary>.
+* The infographic shows <glossary id="anthropogenicgreenhousegases">anthropogenic CO<sub>2</sub> emissions</glossary> and emissions of N<sub>2</sub>O, CH<sub>4</sub>, HFC, PFC, SF<sub>6</sub> and NF<sub>3</sub> expressed in units of CO<sub>2</sub> equivalent (CO<sub>2</sub>eq). This unit takes into account the long-term effect of greenhouse gases in the atmosphere and converts them into an amount of CO<sub>2</sub> that would have the same warming effect. For more, see the article on [Global warming potential](https://en.wikipedia.org/wiki/Global_warming_potential).
+* The left part of the graph shows the absolute emissions of the European Union (these are emissions of the EU-27, i.e. countries that were EU members by 2021) in units of CO<sub>2</sub>eq over the last three decades. The different areas of human activity are distinguished by colour. On the right-hand side, the relative change in emissions compared to 1990 is indicated for each sector.
+* In 1990, emissions in what is now the European Union amounted to 5 billion tonnes of CO<sub>2</sub>eq. In 2019, it was 3.7 billion tonnes. This statistic indicates that EU emissions have fallen by 24% over nearly thirty years. Total EU emissions have been falling more significantly since 2007, after the global financial crisis, in all sectors except transport.
```

Comment:
Change "In 2019, it was 3.7 billion tonnes. This statistic indicates that EU emissions have fallen by 24% over nearly thirty years." to "In 2019, it was 3.7 billion tonnes, indicating that EU emissions have fallen by 24% over approximately thirty years." (To better match tone of Czech original. Also, the "nearly" changes the reader's focus from the point of the sentence.)

Reply:
Thanks, I have updated the text as suggested.